### DISCUSSION TITLE
Products order

### DISCUSSION THREAD

**Discussion by agsmartwolf** [swellstores/community#226] [Help] [answered]
Is there a way to sort a product order?
I fetch all products to show in catalog and I want to display them in specific order.

**Comment by maarten2424** [comment, marked as answer]
Hey @agsmartwolf 
Use the sort parameter to sort results. The format consists of `<field> <direction>`. Direction is recognized as a case-insensitive word like ascending and descending, or as abbreviated asc and desc.
You can combine multiple sort fields in a string or array, with the first taking precedence in order.

With the swell-js library the result would be:
```
await swell.products.list({
  sort: 'name asc',
})
```

**Reply by agsmartwolf** [reply]
Thanks!
I saw that we could drag and drop ordering products in categories. Is there something like that for all of the products?
What way would you recommend to sort them on the dashboard?

I see only one option: create a custom field and manually fill it with integers. (hoping you didn't make a mistake)

**Reply by logeshswell** [reply]
@agsmartwolf I would also suggest you the same option to use the custom integer field that tracks the position of the products.
